<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QMainWindow" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>741</width>
      <height>411</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>741</width>
      <height>491</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="seq">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>420</y>
        <width>164</width>
        <height>32</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>261</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Paramètres</string>
      </property>
      <widget class="QLineEdit" name="time">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="minPos">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>130</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Max Speed</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Max acceleration</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Default Time</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min Pos </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="maxPos">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max Pos </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="speed">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="acc">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>449</x>
        <y>40</y>
        <width>211</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Informations</string>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Taille séquence :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Durée totale :</string>
       </property>
      </widget>
      <widget class="QLabel" name="tailleseq">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="duree">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="moteurs">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>541</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Paramètrage des moteurs</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Moteur 1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Moteur 2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>230</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Moteur 3</string>
       </property>
      </widget>
      <widget class="QComboBox" name="box1">
       <property name="geometry">
        <rect>
         <x>183</x>
         <y>100</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Port :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="option1">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>100</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Option...</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>481</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>481</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>481</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>481</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Port :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="box2">
       <property name="geometry">
        <rect>
         <x>183</x>
         <y>160</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="option2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>160</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Option...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>230</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Port :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="box3">
       <property name="geometry">
        <rect>
         <x>183</x>
         <y>220</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="option3">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>220</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Option...</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="activate1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="activate2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="activate3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>30</y>
        <width>120</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Etat</string>
      </property>
      <widget class="QCheckBox" name="connect1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Déconnecté</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="connect2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Déconnecté</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="connect3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Déconnecté</string>
       </property>
      </widget>
      <widget class="Line" name="line_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="all">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>87</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tout</string>
       </property>
      </widget>
      <widget class="QLabel" name="status">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="info">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Paramètres</string>
    </property>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionCharger"/>
   </widget>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionCharger">
   <property name="text">
    <string>Charger</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
